compiler/passes/normalize.cpp

```diff
@@ -1744,6 +1764,14 @@ static void change_method_into_constructor(FnSymbol* fn) {
     }
   }
 
+  if (ct->instantiationStyle == DEFINES_INITIALIZER) {
+    ArgSymbol* meme = new ArgSymbol(INTENT_BLANK, "meme", ct, NULL,
```

Comment:
The default for records would be CONST_REF which won't make sense for an initializer. For a class, you should set it to CONST_IN since the intent refers to a pointer.